components/brave_ads/content/browser/ad_units/search_result_ad/search_result_ad_util.cc

```diff
@@ -20,6 +21,11 @@ constexpr char kPlacementIdQueryKey[] = "placement_id";
 
 }  // namespace
 
+bool IsPlacementIdValid(const std::string& placement_id) {
```

Comment:
Added helper function to check if placement id is UUID

Reply:
Can we please move this to internal/brave_ads/common as we can reuse this elsewhere. Thanks

Reply:
I think that `IsValidUuid` is too generic and it can be done by code:
```
base::Uuid::ParseCaseInsensitive(placement_id).is_valid()
```

Maybe `IsPlacementIdValid` makes more sense because it is ads related.

Reply:
Ah yes we can use `base::Uuid::ParseCaseInsensitive(placement_id).is_valid()` elsewhere thanks